paper.md

```diff
@@ -0,0 +1,104 @@
+---
+title: 'Hypothesis: A new approach to property-based testing'
+date: 1 November 2019
+bibliography: paper.bib
+tags:
+    - Python
+    - testing
+    - test-case reduction
+    - test-case generation
+    - property-based testing
+authors:
+    - name: David R. MacIver
+      orcid: 0000-0002-8635-3223
+      affiliation: 1
+    - name: Zac Hatfield Dodds
```

Comment:
I tend to use "Hatfield-Dodds" professionally, because using my actual surname with the space is too prone to misinterpretation 🙄. 

(and leads to fairly frequent "invalid name" errors, but that's another story...)